### PR TITLE
Trash

### PR DESCRIPTION
**Pull request by Y2nn1c**
Trash

### CODE REVIEW

File: wiki/Gameplay/Input_device/Mouse/img/mouse_drift_animated.mp4
Comment (Joehuu) [inline wiki/Gameplay/Input_device/Mouse/img/mouse_drift_animated.mp4:1]: I don't think mp4 is supported in wiki (there no mp4 files in this repo), and I believe gif works better here so it repeats.

Comment (Nivalyx) [reply wiki/Gameplay/Input_device/Mouse/img/mouse_drift_animated.mp4:1]: image

Comment (Nivalyx) [reply wiki/Gameplay/Input_device/Mouse/img/mouse_drift_animated.mp4:1]: (no idea if this works on articles but news posts have frequently use youtube embeds)

Comment (Hiviexd) [reply wiki/Gameplay/Input_device/Mouse/img/mouse_drift_animated.mp4:1]: this should ideally be a gif, so we can safely store it in the wiki without worrying about 3rd party sources potentially breaking in the future (i.e. video gets deleted/taken down for whatever reason).

Comment (cl8n) [reply wiki/Gameplay/Input_device/Mouse/img/mouse_drift_animated.mp4:1]: > (no idea if this works on articles but news posts have frequently used youtube embeds)

html is disabled in wiki articles

**Comment by peppy** [comment]
Neither of these diagrams make any sense to my brain. I checked for a second opinion and they also couldn't follow.

**Comment by cl8n** [comment]
I agree the static image is hard to understand, I would scrap that. the animated one gets the idea across to me, but it's a bit abstract, and also the aesthetics clash hard with the wiki. I kinda want to try making an animation that shows this same concept but literally using osu to explain it

**Pull request closed without merging**